main.go

```diff
@@ -6,17 +6,18 @@ import (
 	"github.com/gin-contrib/cors"
 	"github.com/gin-gonic/gin"
 	"github.com/iver-wharf/wharf-api-client-go/pkg/wharfapi"
+	_ "github.com/iver-wharf/wharf-provider-gitlab/docs"
 	log "github.com/sirupsen/logrus"
 	swaggerFiles "github.com/swaggo/files"
 	ginSwagger "github.com/swaggo/gin-swagger"
-	_ "github.com/iver-wharf/wharf-provider-gitlab/docs"
 )
 
-// BUILD_DEFINITION_FILE_NAME is a name of the file that should exists in project repository if project will have ability to be built by Wharf.
-const BUILD_DEFINITION_FILE_NAME = ".wharf-ci.yml"
+// BuildDefinitionFileName is a name of the file that should exists in project
+// repository if project will have ability to be built by Wharf.
+const BuildDefinitionFileName = ".wharf-ci.yml"
 
-// PROVIDER_NAME is a provider name that is used in whole wharf system for GitLab.
-const PROVIDER_NAME = "gitlab"
+// ProviderName is a provider name that is used in whole wharf system for GitLab.
+const ProviderName = "gitlab"
```

Comment:
Should we not change the linting to instead allow for what seems like the java equivilant of `static final` to be in SNAKE_CASE? Or perhaps the linting is warning because its missing a modifier that could/should be there?

Reply:
Nah upper-cased snake case is used in C# and Java, but the convention in Go is to only use camel- and pascal casing

We concluded around half a year ago that we want to follow the CodeReviewComments and Effective Go guides for future code, so it's more "conventional Go code" and as we were all noobs in Go we didn't have the energy to seek our own conventions

https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

Reply:
And this variable was named before we made that descision.